spec/services/district_report_service_spec.rb

```diff
@@ -94,19 +97,19 @@ def refresh_views
 
     refresh_views
 
-    service = DistrictReportService.new(facilities: facility_group_1.facilities, selected_date: june_1, current_user: user)
+    service = DistrictReportService.new(district: facility_group_1, selected_date: june_1, current_user: user)
     result = service.call
 
     expected_controlled_patients = {
       "Jan 2019" => 2, "Feb 2019" => 2, "Mar 2019" => 2, "Feb 2020" => 2, "Mar 2020" => 2, "Apr 2020" => 4, "May 2020" => 2, "Jun 2020" => 2
     }
     expected_controlled_patients.default = 0
     expected_registrations = {
-      "Jan 2020" => 4, "Feb 2020" => 4, "Mar 2020" => 6, "Apr 2020" => 6, "May 2020" => 6, "Jun 2020" => 6
+      "Dec 2018" => 0, "Jan 2020" => 4, "Feb 2020" => 4, "Mar 2020" => 6, "Apr 2020" => 6, "May 2020" => 6, "Jun 2020" => 6
     }
     expected_registrations.default = 2
-    expect(result[:controlled_patients].size).to eq(18)
-    expect(result[:registrations].size).to eq(18)
+    # expect(result[:controlled_patients].size).to eq(19)
+    # expect(result[:registrations].size).to eq(19)
```

Comment:
We can trash these assertions if we're not using them.

Reply:
Fixed these and added them back.